pkg/universe.dagger.io/alpha/uffizzi/run.cue

```diff
@@ -0,0 +1,75 @@
+package uffizzi
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/docker"
+	"universe.dagger.io/bash"
+)
+
+#Run: {
+	source: dagger.#FS
+
+	uffizzi_user:     string
+	uffizzi_password: dagger.#Secret
+	uffizzi_project:  *null | string
+	uffizzi_server:   string
+
+	docker_compose_file: *null | string
+	deployment_id:       *null | string
+
+	dockerhub_username: string
+	dockerhub_password: dagger.#Secret
+
+	// Entity on which the command is being performed eg: project, preview etc
+	entity: string
+	// Verb, action to perform based on the given entity
+	verb: string
+
+	// Environment variables
+	// env: [string]: string | dagger.#Secret  
+
+	_source_path: "/src"
+	version:      *"latest" | string
+
+	_image: #Image
+
+	bash.#Run & {
+		input: *_image.output | docker.#Image
+		script: {
+			_load: core.#Source & {
+				path: "."
+				include: ["*.sh"]
+			}
+			directory: _load.output
+			filename:  "uffizzi.sh"
+		}
+
+		always: true
```

Comment:
This field should always be optional (so you can remove it), it's up to the final user to enable or disable cache.

Reply:
Understood.